src/main/kotlin/com/ampnet/userservice/service/pojo/IdentyumData.kt

```diff
@@ -34,13 +34,13 @@ data class Data(
 data class PersonalData(
     val firstName: ValueSource,
     val lastName: ValueSource,
-    val nationalityCode: ValueSource,
+    val nationalityCode: ValueSource?,
     val dateOfBirth: ValueSource,
-    val sex: ValueSource,
-    val adresses: List<ValuesSources>,
-    val personalNumbers: List<ValuesSources>,
-    val emails: List<ValuesSources>,
-    val phones: List<ValuesSources>,
+    val sex: ValueSource?,
+    val adresses: List<ValuesSources>?,
```

Comment:
typo rename to: addresses

Reply:
It is a typo in Identyum response if we change it, mapping won't work

Reply:
Ok :)